server/rest/annotation.py

```diff
@@ -160,6 +161,30 @@ def createAnnotation(self, item, params):
                     exc.args, ))
 
     @describeRoute(
+        Description('Copy an annotation from one item to an other.')
+        .param('annotationId', 'The ID of the annotation.', paramType='path',
+               required=True)
+        .param('itemId', 'The ID of the destination item.',
+               required=True)
+        .errorResponse('Write access was denied for the item.', 403)
+        .errorResponse('Validation Error: JSON doesn\'t follow schema.')
+    )
+    @access.user
+    @loadmodel(model='annotation', plugin='large_image')
+    @filtermodel(model='annotation', plugin='large_image')
+    def copyAnnotation(self, annotationId, itemId, params):
+        user = self.getCurrentUser()
+        annotation = self.model('annotation', 'large_image').load(
+            annotationId,
+            level=AccessType.READ,
+            region=params
+        )
```

Comment:
The annotation model is not access controlled, so the `level` argument here doesn't work.  You will need to validate read access to the source item, probably by loading it with the correct keyword arguments.

Reply:
Ok I was not sure, thank you